model_analyzer/analyzer.py

```diff
@@ -118,12 +118,17 @@ def profile(self, client):
             f"Finished profiling. Obtained measurements for models: {profiled_model_list}."
         )
 
-    def analyze(self):
+    def analyze(self, mode):
```

Comment:
I don't think `mode` is required for `analyze`.

Reply:
It is. There are different set of presets and parameters depending on mode.

Reply:
Yes, I understood that. But I think `mode` doesn't change anything in the `analyze` mode so I was wondering why it is required.

Reply:
It changes the summary a little bit. Shows different plots.